app-text/cherrytree/cherrytree-0.99.0_p20200520.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+
+HASH_COMMIT="4e66c82c108bdfb56983d2c3614d88cc61f5179e" # 20200520
+
+inherit cmake python-any-r1 xdg-utils
+
+DESCRIPTION="A hierarchical note taking application (C++ version)"
+HOMEPAGE="https://www.giuspen.com/cherrytree"
```

Comment:
Homepage appears to be down, can you verify that? Maybe we should add their github homepage and revisit this on next bump to drop it if this is permanent.

Reply:
https://www.giuspen.com/cherrytree — works fine for me